docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -157,7 +157,7 @@ The operation was successful if the returned status says `true`.
 
 ## Trigger the workload with an event
 
-We created the `lastorder` Function, and created a Subscription for it to listen for `order.received.v1` events. Now it's time to send an event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost. 
+We created the `lastorder` Function, and subscribed for `order.received.v1` events by creating a Subscription CR. Now it's time to publish an event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost. 
```

Comment:
```suggestion
We created the `lastorder` Function and subscribed to the `order.received.v1` event by creating a Subscription CR. Now it's time to publish your event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost. 
```

Reply:
We're talking about one event here, is that correct?

Reply:
yes